### DISCUSSION TITLE
Sonoff basic keeps disconnecting and reconnecting to wifi

### DISCUSSION THREAD

**Discussion by BrutoCane** [arendst/Tasmota#20275] [Q&A]
Hi, i have two sonoff basic r1 and im having some trouble connecting to wifi
I installed tasmota (from 5.14 to 13.3) and when i power on the sonoff i can connect to his AP.
I tried every possible combination, 2.4/5Ghz, automatic, guests wifi, WPA2, WPA3, no psw.. but the device keeps connect and disconnecting from the wifi
I also uploaded a video on youtube
https://youtu.be/-n1xiDRuw2I
my router is a asus rog ax6000
Can you help me? Thanks

**Comment by barbudor** [comment]
Did you searched a little before asking?
This has already been answered many times in the past months
For this hardware, use tasmota-lite
It's still a bit slow to respond but it works

**Reply by BrutoCane** [reply]
Yeah i searched but found nothing, even on the official website
i tought the problem was the ssid with blank space in it, but i was wrong
im going to try tasmota-lite, thanks

**Reply by sfromis** [reply]
Also, routers from Asus are common trouble-makers. I'm not sure which models, or if it is a likely cause.

**Reply by BrutoCane** [reply]
i think it's working
with tasmota-lite 13.3 i cant use ssid with white space
But tasmota 9.3 seems to be working

Thank you both!

**Comment by BrutoCane** [comment]
One now is working fine, the other one is still having the same problem
now i tried installing tasmota-lite 9.5 but it keep saying "Connect failed, please check your credentials"
But if i check the log on the router it says:
"Dec 21 11:06:56 wlceventd: wlceventd_proc_event(530): eth6: Auth "MAC", status: Successful (0), rssi:-67
Dec 21 11:06:56 wlceventd: wlceventd_proc_event(559): eth6: Assoc "MAC", status: Successful (0), rssi:-67"

**Comment by Jason2866** [comment]
Once again search here in discussions for Asus Issues.
